lib/davinci_dtr_test_kit/dtr_light_ehr_suite.rb

```diff
@@ -62,12 +46,12 @@ class DTRLightEHRSuite < Inferno::TestSuite
     ]
 
     input :url,
-          title: 'FHIR Endpoint',
-          description: 'URL of the DTR FHIR server'
+          title: 'FHIR Server Base Url',
+          description: 'Required for All Flows'
```

Comment:
"All Flows" is something from the Payer server tests and doesn't apply here. I think change it back to something more descriptive, like "URL of the target DTR Light EHR".